File: C++/Step2-If/1330.cpp
//
//  1330.cpp
//  Step2-If
//
//  Created by 김예빈 on 2019. 8. 28..
//  Copyright © 2019년 김예빈. All rights reserved.
//

#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    int a, b;
    
    cin >> a >> b;
    
    if (a > b)
        cout << ">";
    else if (a < b)
        cout << "<";
    else
        cout << "==";
    
    return 0;
}

File: BaekJoon Online Judge/Swift/Step02-If/1330.swift
//
//  1330.swift
//  Step02-If
//
//  Created by Yebin Kim on 2019/10/27.
//  Copyright © 2019 김예빈. All rights reserved.
//

import Foundation

var input = readLine()

if let input = input {
    let arr = input.components(separatedBy: " ")
    
    let A = Int(arr[0])!
    let B = Int(arr[1])!
    
    if(A < B) {
        print("<")
    } else if(A > B) {
        print(">")
    } else {
        print("==")
    }
}
